src/utils/parse-text.js

```diff
@@ -127,6 +140,23 @@ const redditLinks = () => {
   });
 };
 
+const shortLinks = () => {
+  const beforeChars = new RegExp(`[${wordAdjacentChars}]`);
+  const afterChars = new RegExp(`[${wordAdjacentChars.clone().removeChars('/')}]`);
```

Comment:
I think `//clbn/a0f76b` should NOT be hyperlinked the same way `/clbn/a0f76b/` isn't. (So I would add `removeChars('/')` to `beforeChars` as well.)

Reply:
Well, it will be linked as `/[/clbn/a0f76b]`, which isn't too bad... Maybe there are some specific cases in which this can lead to incorrect results?

Reply:
Ok, I'd fixed it.